### PR TITLE
Feature/#3176 implement user's order details page

### PR DESCRIPTION
**Pull request by KhrystynaPr**
added logic to receive all orders for logged user and details of these orders

### CODE REVIEW

**Review by dyadyushko — COMMENTED** [review]
something happend to your packege-lock.json, we should not transfer those changes into dev branch

File: src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  <table class="table_of_details">
+    <tr class="header_table">
+      <th>{{ 'user-orders.details.services' | translate }}</th>
+      <th>{{ 'user-orders.details.volume' | translate }}</th>
+      <th>{{ 'user-orders.details.cost' | translate }}</th>
+      <th>{{ 'user-orders.details.bags-amount' | translate }}</th>
+      <th>{{ 'user-orders.details.sum' | translate }}</th>
+    </tr>
+    <tr *ngFor="let bag of order.bags" class="data_table">
+      <td>{{ bag.name }}Вторсировина</td>
```
Comment (nazarf1) [inline src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.html:12]: do we need "вторсировина" here?

File: src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  templateUrl: './ubs-user-order-details.component.html',
+  styleUrls: ['./ubs-user-order-details.component.scss']
+})
+export class UbsUserOrderDetailsComponent implements OnInit {
+  @Input()
+  order: any;
+
+  constructor() {}
+
+  ngOnInit() {}
```
Comment (nazarf1) [inline src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.ts:14]: empty,remove

**Review by NewArtMask — APPROVED** [review]

File: src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.html (hunk 1 above)
Comment (KhrystynaPr) [reply src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.html:12]: yes, because we don`t have this data in the backend yet

**Review by s1auka — APPROVED** [review]

File: src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.ts (hunk 2 above)
Comment (KhrystynaPr) [reply src/app/ubs-user/ubs-user-order-details/ubs-user-order-details.component.ts]: done

**Review by nazarf1 — APPROVED** [review]

File: src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <div *ngIf="order.orderStatus === 'DONE_UNPAID'" class="btn_grid_fake">
+        <button class="btn_fake"></button>
+      </div>
+      <div *ngIf="order.orderStatus === 'FORMED' || order.orderStatus === 'DONE_UNPAID'" class="btn_grid_pay">
+        <button class="btn_pay">{{ 'user-orders.btn.pay' | translate }}</button>
+      </div>
+      <div
+        *ngIf="order.orderStatus === 'ON_THE_ROUTE' || order.orderStatus === 'CONFIRMED' || order.orderStatus === 'DONE'"
+        class="btn_grid_order"
+      >
```
Comment (Edw1x) [inline src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.html:26]: Move this to ts file and make getters
```
get orderFormed() {
    return order.orderStatus === 'FORMED'
}
```
*ngIf="orderFormed"

File: src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+@Component({
+  selector: 'app-ubs-user-orders-list',
+  templateUrl: './ubs-user-orders-list.component.html',
+  styleUrls: ['./ubs-user-orders-list.component.scss']
+})
+export class UbsUserOrdersListComponent {
+  @Input()
+  orders: any[];
+
+  constructor() {}
```
Comment (Edw1x) [inline src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts:12]: empty

File: src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+  changeCard(id: number) {
+    this.orders.map((el) => {
+      el.id === id && el.extend ? (el.extend = false) : el.id === id ? (el.extend = true) : (el.extend = el.extend);
```
Comment (Edw1x) [inline src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts:16]: It's better to use understandable variable
Suggestions:
'order'

**Review by nazarhlyvka — APPROVED** [review]

File: src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.html (hunk 3 above)
Comment (KhrystynaPr) [reply src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.html]: done

File: src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts (hunk 4 above)
Comment (KhrystynaPr) [reply src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts]: done

File: src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts (hunk 5 above)
Comment (KhrystynaPr) [reply src/app/ubs-user/ubs-user-orders-list/ubs-user-orders-list.component.ts]: done

**Pull request merged**